conan/tools/microsoft/msbuild.py

```diff
@@ -16,10 +22,21 @@ def msbuild_arch(arch):
             'armv8': 'ARM64'}.get(str(arch))
 
 
+def msbuild_arch_to_conf_arch(arch):
```

Comment:
If used only once, then it is better to keep it local initially, until repetition pattern clearly arises (ideally at least 3 occurrences)

Reply:
It's not exposed in `__init__.py` of `microsoft` folder, so it's not public according to conan documentation. What should I do? There are other non-public functions in this file actually.